nomenclature/definitions/variable/fuel_types.yaml

```diff
@@ -126,5 +132,25 @@
   Ocean:
     description: Ocean energy
 
+  Electricity Storage:
```

Comment:
Isn't it better to use `Capacity|Electricity Storage:` instead of? We already use `Maximum Storage|Electricity|Energy Storage System:` in `power-plan.yaml` to point out the same definition. Thus, I suggest continuing using `Electricity|Energy Storage System` for all variables related to it.

Reply:
Sorry, but I'm not 100 percent clear about your comment: Do you want me to remove the `Electricity Storage` in this yaml file? The auto-expansion creates, for example, `Capacity|Electricity|Electricity Storage` as I removed the original `Capacity|Electricity Storage` from the technologies.yaml file. Also, I want to point out, that I did this change because `Capacity|Electricity|Electricity Storage` was required by Deliverable 5.2 as a variable from GENeSYS-MOD.

Reply:
From my perspective, `Capacity|Electricity|Electricity Storage` is redundant; that is why we decided to use `Maximum Storage|Electricity|Energy Storage System:` that means the maximum capacity to store electricity in an ESS. We can also use the base `Energy Storage System` for any energy (e.g., electricity, gas, or heat).

Reply:
Ah ok, I always thought of the variables in `power-plant.yaml` as variables for defining "typical" storage or power plant types, whereas in `technologies.yaml` are the variables for aggregated results:

E.g., `Maximum Storage|Electricity|Energy Storage System|Lithium-Ion` defines how much energy can be stored within a typical Lithium-Ion battery storage. Contrary, `Capacity|Electricity|Electricity Storage|Lithium-Ion` defines how much energy can be stored in 2050 in a whole country. But this really might be a misunderstanding from my side.

Reply:
I think that's no problem because, for each variable, we have to define the region (e.g., related a whole country, a specific region, or NUTS), and if it is related to a time-series, a yearly value, or sub-annual value.

Reply:
I removed the electricity storages from the `fuels.yaml`. However, Pumped Hydropower storages are also prevalent in both places (in `fuels.yaml` and `power_plants.yaml`) and I didn't dared to touch those, as there were defined as such beforehand.